BlankSolution/HW06.Task3/Program.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Diagnostics;
+
+namespace HW06.Task3
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            int[] arr = new int[5] { 1, 2, 3, 4, 5 };
+
+            for (int i = 0; i < (arr.Length - 1)/2; i++)
+            {
+                int firstValue = arr[i];
+                arr[i] = arr[arr.Length - 1 - i];
+                arr[arr.Length - 1 - i] = firstValue;
+            }
+
+            for (int i = 0; i < arr.Length; i++)
+            {
+                Console.Write(arr[i] + " ");
+            }
+
+            Console.WriteLine();
+
+            //Testing
+            long[] testArr = new long[1000000000];
```

Comment:
you can also use the following literal declaration for more flexibility:

`new long[1000000000]`  = `new long[1_000_000_000]`